app/code/community/Drip/Connect/Model/ApiCalls/Helper.php

```diff
@@ -22,13 +22,6 @@ abstract class Drip_Connect_Model_ApiCalls_Helper
     /** @var Drip_Connect_Model_ApiCalls_Request_Base */
     protected $request;
 
-    /**
-     * must be implemented individually for every call helper
-     *
-     * two classes should be instantiated in every constructor: ApiClient and Request
-     */
-    abstract public function __construct($data);
```

Comment:
Since we aren't using `getModel` anymore, each class has different parameters. So this abstract declaration is problematic.